src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -747,6 +747,33 @@ export class PostgresStorageAdapter {
       });
   }
 
+  schemaUpgrade(className, schema) {
+    debug('schemaUpgrade', { className, schema });
+
+    return this._client.tx('schemaUpgrade', t => {
+      return t.any('SELECT column_name FROM information_schema.columns WHERE table_name = $<className>', { className })
+        .then(columns => {
+          const currentColumns = columns.map(item => item.column_name);
```

Comment:
So I see this is the only bit you didn't change here. As is this is passing our tests and looks good. Perhaps it's fine as is. @vitaly-t do you think this is important enough to change or are we good as is?

Reply:
@vitaly-t Ding? Zing? Ping?

Reply:
This code should be replaced with ES6 generators, the way it was done for all tasks and transaction in the code.